tradingstrategy/testing/mock_client.py

```diff
@@ -41,6 +37,35 @@ def fetch_candle_dataset(self, bucket: TimeBucket):
 
     def fetch_all_liquidity_samples(self, bucket: TimeBucket) -> Table:
         raise MockClientNotImplemented()
+    
+    def get_default_quote_token_address(self, factory_address: str | None = None) -> str:
+        """Get the quote token address used in the generated pair map.
+
+        Helper method for setting up simple local dev routing.
+
+        Returns a the first quote token address found in the pair universe.
+
+        :param factory_address:
+            The factory address to get the quote token address from.
+        """
+        quote_tokens = []
+        pairs_df = self.fetch_pair_universe().to_pandas()
+        pair_universe = PandasPairUniverse(pairs_df)
+        assert pair_universe.get_count() > 0, "Pair universe has no trading pairs"
+        for pair in pair_universe.iterate_pairs():
+            if factory_address:
+                if pair.exchange_address.lower() == factory_address.lower():
+                    quote_tokens.append(pair.quote_token_address)
+            else:
+                quote_tokens.append(pair.quote_token_address)
+        #assert len(quote_tokens) == 1, f"Got {len(quote_tokens)} quote tokens in the pair universe, the pair universe is total {pair_universe.get_count()} pairs"
+        return quote_tokens[0]
+
+    def fetch_exchange_universe(self) -> ExchangeUniverse:
```

Comment:
`fetch` verrb is for reading something from an external source. Here it just returns an value. There is no need to create getter, as it is just a simple attribute.

Reply:
I just extracted this function as is from `UniswapV2MockClient`. It is also used in quite a few places in tradingstrategy and tradeexecutor. Should I still change it?